### PR TITLE
update default Flow origin to {clientLabel}.formsort.app

### PR DESCRIPTION
**Pull request by atikenny**
Implements [FLOW-150](https://formsort.atlassian.net/browse/FLOW-150)

Use `{clientLabel}.formsort.app` when origin is not specified.

### CODE REVIEW

File: packages/web-embed-api/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.2.1
+
+- Use `{clientLabel}.formsort.app` if no origin was specified. `flow.formsort.com/{clientLabel}` is deprecated.
```
Comment (BYK) [inline packages/web-embed-api/CHANGELOG.md:5]: Revert this change please, the changelog should be auto-generated.

Comment (atikenny) [reply packages/web-embed-api/CHANGELOG.md:5]: What will it be based on?

**Review by BYK — CHANGES_REQUESTED** [review]
Would have been better if we separated the package updates from the main change. Rejecting due to the unused variable warning and the changelog entry.

**Review by BYK — APPROVED** [review]
